packages/airnode-adapter/src/response-processing/encoding.ts

```diff
@@ -21,12 +21,20 @@ export function getSolidityType(type: ResponseType) {
 
 export function encodeValue(value: ValueType, type: ResponseType): string {
   const solidityType = getSolidityType(type);
-
-  return ethers.utils.defaultAbiCoder.encode([solidityType], [value]);
+  try {
+    const encodedValue = ethers.utils.defaultAbiCoder.encode([solidityType], [value]);
+    return encodedValue;
+  } catch (e) {
+    throw new EncodingError((e as Error).message);
```

Comment:
How certain are we that we can cast e as Error? Should we check it's type first like in [here](https://github.com/api3dao/airnode/blob/v1-protocol/packages/airnode-admin/src/cli.ts#L479-L480)?